qiskit/transpiler/preset_passmanagers/common.py

```diff
@@ -447,9 +446,7 @@ def generate_translation_passmanager(
     Raises:
         TranspilerError: If the ``method`` kwarg is not a valid value
     """
-    if method == "unroller":
```

Comment:
With this removal, we also should remove the corresponding plugin, right? 

https://github.com/Qiskit/qiskit/blob/c799435fc1cc443e4701793930e68f7a07c7aa22/qiskit/transpiler/preset_passmanagers/builtin_plugins.py#L179

Reply:
Yes that is correct, I will do that.